src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -1,14 +1,21 @@
-import { Typography, Grid, Card, Box } from '@mui/material'
-import { WidgetBody } from '@/components/dashboard/styled'
+import { Typography, Grid, Card, Box, Accordion, AccordionSummary, AccordionDetails } from '@mui/material'
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
-import Accordion from '@mui/material/Accordion'
-import AccordionSummary from '@mui/material/AccordionSummary'
-import AccordionDetails from '@mui/material/AccordionDetails'
+import { useState } from 'react'
+import type { SyntheticEvent } from 'react'
+
+import SnapshotWidget from '../SnapshotWidget'
+
 import css from './styles.module.css'
 
-const GovernanceSection = () => (
-  <Grid item xs={12} md>
-    <Accordion className={css.accordion}>
+const GovernanceSection = () => {
+  const [expanded, setExpanded] = useState(true)
+
+  const handleChange = (_: SyntheticEvent, isExpanded: boolean) => {
+    setExpanded(isExpanded)
+  }
```

Comment:
The state and the onClick handler are already handled by the `<Accordion />`

Reply:
I've removed the state.